include/util/debug_geometry.hpp

```diff
@@ -69,6 +71,8 @@ boost::filesystem::ofstream dg_debug_turns_file;
 bool dg_output_turn_debug = false;
 bool dg_first_turn_debug = true;
 
+std::unordered_map<OSMNodeID, util::FixedPointCoordinate> node_lookup_map;
```

Comment:
Shouldn't need `static` in this case.  AFAIK, `static` on global variables doesn't turn them into singletons, that only applies for function-scoped and class-member static stuff.